### PR TITLE
Jdiazdelrio pr202 conda fix

### PR DESCRIPTION
**Pull request by AndrewAnnex**

### CODE REVIEW

**Comment by AndrewAnnex** [comment]
This may be a problem: https://github.com/conda-forge/selenium-feedstock/issues/16 ie conda-forge is dropping 3.4 support. So I would have to maintain my own certifi deployment potentially or wait for anaconda to update or tie to a lower certifi version that is available through official channels

**Comment by AndrewAnnex** [comment]
@jdiazdelrio I had to try a lot of things in order to get python 34 to builds to work, it appears that the main issue is that anaconda/conda-forge are starting to drop 34, so I had to go to an older (still 2017) release of certifi. Take a look at the diff and let me know what you think

That all being said, this problem is basically solved if we drop 34 support. I have no particular need for 34 support other than I was asked about continuing to support  3.4 earlier in the year #181 and earlier #168. Also after going through all of the trial and error I am sort of interested in possibly making the urllib3 pathway the default behavior to remove the confusion for conda/pip for when pyopenssl and such is needed. I am sure this contradicts opinions I have said before but the code smell from the conda/appveyor configs is starting to get to me.

**Comment by jdiazdelrio** [comment]
@AndrewAnnex It looks fine. Using an older version of certifi when no newer version is available will do the job. 

Regarding dropping 3.4, I would not do it. conda may drop 3.4, but Python3.4 will still be around for about 18 months (mar-2019 it's the end-of-life, I think). Maybe for the next SPICE toolkit release Python3.4 is no longer around and then it would be justifiable. 

About making urrlib3 the default pathway, I would be in favor, at least for the time being, since it's always better doing the same for all supported versions, but that's a programmers choice :)

**Comment by AndrewAnnex** [comment]
@jdiazdelrio ok sounds good. how about you update your pr to match the needed changes I made in this branch (then I will close #232 ). Then I can merge in your PR.

**Comment by jdiazdelrio** [comment]
@AndrewAnnex Alright. I will do that tomorrow. Besides the changes to the AppVeyor configuration, and the changes in the certifi version, would you like that I implement your suggestion of putting urllib3 as default pathway?

Am I missing anything else?

**Comment by AndrewAnnex** [comment]
I think there a few comments unaddressed in #202 but they are all minor. The urllib3 only idea can wait for a bit or I may make the change later.

File: setup.py [hunk 1]
```diff
@@ -21,10 +21,13 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
+from __future__ import print_function
```
Comment (AndrewAnnex) [inline setup.py]: [picky] is this import needed?

**Pull request closed without merging**